src/helpers.py

```diff
@@ -139,7 +139,7 @@ def request(url, method, enable_5xx=False, payload=None):
     if method == "GET" or not method:
         res = r.get("%s" % url, auth=auth, timeout=REQ_TIMEOUT, verify=REQ_TLS_VERIFY)
     elif method == "POST":
-        res = r.post("%s" % url, auth=auth, json=payload, timeout=REQ_TIMEOUT, verify=REQ_TLS_VERIFY)
+        res = r.post("%s" % url, auth=auth, json=json.loads(payload), timeout=REQ_TIMEOUT, verify=REQ_TLS_VERIFY)
```

Comment:
I think the fix is valid. However, the json parsing should be done initially when loading from the environment, not each time a request is being made.
Also - to be backwards compatible - when parsing fails we need to fall back to the current behavior.

Reply:
fair enough, I updated with a method doing the conversion and a fallback.